doc/man3/EVP_PKEY_decapsulate.pod

```diff
@@ -30,14 +30,29 @@ key that is used during decapsulation.
 
 The EVP_PKEY_decapsulate() function performs a private key decapsulation
 operation using I<ctx>. The data to be decapsulated is specified using the
-I<wrapped> and I<wrappedlen> parameters.
-If I<unwrapped> is NULL then the size of the output secret buffer
-is written to I<*unwrappedlen>. If I<unwrapped> is not NULL and the
-call is successful then the decapsulated secret data is written to I<unwrapped>
-and the amount of data written to I<*unwrappedlen>.  Note that, if I<unwrappedlen>
-is not NULL in this call, the value it points to must be initialised to the length of
-I<unwrapped>, so that the call can validate it is of sufficient size to hold the
-result of the operation.
+I<wrapped> and I<wrappedlen> parameters (which must both non-NULL).
```

Comment:
This looks like a change in documented behavior. Was it always like this, is did it change?

Reply:
Is this then something that should be back ported to other branches?

Reply:
Well, the documentation is shared.
There's not anything specific to "backport" as such, but sure the other providers can be (new code) reviewed and improved to be resilient in the face of NULL values for the in/out length pointers.  Nothing in ML-KEM or ML-DSA is something to backport, more *emulate* I guess. And an issue can  be opened to track that, if one hasn't been already.

Reply:
How is the documentation shared? Distributions ship the documentation as is in the branch, and people will read wrong documentation.

Reply:
This should really be handled one way or another after this PR is merged. IMO there should be a separate issue if we do not have one yet. @kroeckx could you please create a new issue?

Reply:
Sorry, the documentation is shared across the providers in 3.5, it is of course not shared with 3.4 or earlier.
So I guess we could backport the docs, or make the providers more resilient across all algorithms and backport that as appropriate. Anyway, not much to do in this PR.